gammapy/datasets/tests/test_simulate.py

```diff
@@ -590,6 +590,43 @@ def test_mde_run(dataset, models, tmp_path):
     assert u.isclose(obs_back.observatory_earth_location.height, LOCATION.height)
 
 
+@requires_data()
+def test_sampler_output(dataset, model_alternative, caplog):
```

Comment:
Probably it's better to add the testing of the logging output to one of the existing test. Sampling a whole new event list just for testing the log output is a bit too much.